File: internal/indexer/testdata/java/views/UserViews.java
package views;

import static services.UserService.processUserData;
import static services.UserService.validateEmail;
import services.UserService;

import java.util.Map;

public class UserViews {
    public boolean handleUpdate(int userId, Map<String, String> data) {
        boolean result = processUserData(userId, data);
        return result;
    }

    public boolean handleValidate(String email) {
        return validateEmail(email);
    }

    public String handleFormat(String first, String last) {
        return UserService.formatUserName(first, last);
    }
}

File: internal/indexer/testdata/go/views/user_views.go
package views

import "services"

func HandleUpdate(userID int, data map[string]string) bool {
	result := services.ProcessUserData(userID, data)
	return result
}

func HandleValidate(email string) bool {
	return services.ValidateEmail(email)
}

func HandleFormat(first, last string) string {
	return services.FormatUserName(first, last)
}
